RecoLocalCalo/CaloTowersCreator/src/CaloTowersCreator.cc

```diff
@@ -62,7 +63,8 @@ CaloTowersCreator::CaloTowersCreator(const edm::ParameterSet& conf) :
         conf.getParameter<double>("MomHBDepth"),
         conf.getParameter<double>("MomHEDepth"),
         conf.getParameter<double>("MomEBDepth"),
-        conf.getParameter<double>("MomEEDepth")
+        conf.getParameter<double>("MomEEDepth"),
+        conf.exists("HcalPhase") ? conf.getParameter<int>("HcalPhase") : 0
```

Comment:
use of exists is discouraged.
Please add fillDescriptions method and define the default value there